### PR TITLE
Leaderboard: setup project

### PR DESCRIPTION
**Pull request by gracekabaghe**
- Project setup with HTML, CSS, Webpack

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (D4nielJ) [inline .gitignore:1]: - [x] Your `node_modules` folder has been uploaded to GitHub. Let's remove the folder from the repository and update `.gitignore` to ignore it correctly. [You can read how to do it without removing the files from your local environment here.](https://stackoverflow.com/questions/1143796/remove-a-file-from-a-git-repository-without-deleting-it-from-the-local-filesyste)

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @gracekabaghe, 

You're doing great! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (gracekabaghe) [reply .gitignore:1]: Thanks for sharing the tutorial on how to solve this. I have removed node modules

**Review by o-rashad — APPROVED** [review]
Asslam Alaikum @gracekabaghe  ,😊

Hi👋🏽, Your project is complete!  There is nothing else to say other than... it's time to merge it 😎 

# Congratulations! 🎉 

Happy coding!👏

## Notes

As this is only the first setup, we can ignore the nonexistence of the `README` file,
> - [ ] [OPTIONAL] Be sure to update your README with the project's information, so anybody who looks at your portfolio will be able to tell what the project is about. [Remember that this is the expected structure for a Microverse Project's README.](https://github.com/microverseinc/readme-template)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**